Components/Graphic/index.tsx

```diff
@@ -0,0 +1,69 @@
+import { Flex } from "@chakra-ui/react";
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Filler,
+  Legend,
+} from "chart.js";
+import { Line } from "react-chartjs-2";
+import { useMemo, useState } from "react";
+import type { IItems } from "../../pages/price";
+
+ChartJS.register(
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Title,
+  Tooltip,
+  Filler,
+  Legend
+);
+export const options = {
+  responsive: true,
+  plugins: {
+    legend: {
+      position: "top" as const,
+    },
+    title: {
+      display: true,
+      text: "Preço semanal",
+    },
+  },
+};
+export const Graphic = () => {
```

Comment:
Esse componente não precisa de useStates, ele vai receber o currentItem e apresentar os dados dele

Reply:
Cria uma interface aqui pra dizer o que ele recebe:
```interface GraphicProps {
currentItem: IItem

}```

Reply:
Como eu pego currentItem da price ?

Reply:
> Cria uma interface aqui pra dizer o que ele recebe:
> 
> ```
> currentItem: IItem
> 
> }```
> ```
eu passo essa tipagem pro component graphic?

Reply:
```export const Graphic: React.FC<ICurrentItem> = (prop: ICurrentItem) =>``` tipei assim mas ainda não to achando currentItem mesmo assim

Reply:
Essa props.currentItem eu passo no param no component?

Reply:
Você diz que na hora de pegar os currentItem eu pego props.currentItem ?  eu troquei os currentItem por props.currentItem e quando eu clica retorna 12 undefined no console.log

Reply:
da um console log no props pra ver